cfgov/jinja2/v1/_includes/organisms/video-player.html

```diff
@@ -127,6 +127,11 @@
                         {{- ' o-featured-content-module_img' if is_fcm else '' }}"
                  alt=""
                  src="{{ image_url }}">
+
+            {#
+                Provide a duration timestamp in the corner of the video.
+            #}
+            <div class="o-video-player_duration"></div>
```

Comment:
Should this be a `<time>` element with a [duration string](https://www.w3.org/TR/2014/REC-html5-20141028/infrastructure.html#valid-duration-string) ([e.g.](https://stackoverflow.com/a/31591343))?

Reply:
Converted this to a `<time>` element. For the `datetime` string I turned it into `###S`, e.g. `159S`. Do you agree that appears to be correct according to the second bullet on https://www.w3.org/TR/2014/REC-html5-20141028/infrastructure.html#valid-duration-string ?

Reply:
Smart! 👍